### PR TITLE
Add accessibility label to feedback list view cell

### PR DESCRIPTION
**Pull request by erychagov**
I found only one place where screen reader didn't read all information.

### CODE REVIEW

File: Resources/ja.lproj/HockeySDK.strings [hunk 1]
```diff
@@ -274,3 +274,6 @@
 "HockeyAuthenticationContactDeveloper" = "認証エラー。このAppのデベロッパに連絡してください。";
 "HockeyAuthenticationWrongEmailPassword" = "無効なメールアドレスまたはパスワードが入力されました。入力を確認し、やり直してください。";
 "HockeyAuthenticationAuthSecretMissing" = "HockeySDK “authorization secret” が設定されていません。このAppのデベロッパに連絡してください。";
+
+/* Accessibility strings */
+"HockeyAccessibilityHintRequired" = "必須";
```
Comment (Huan9) [inline Resources/ja.lproj/HockeySDK.strings]: "HockeyAccessibilityHintRequired" = "必填"; to be consistent with https://github.com/bitstadium/HockeySDK-iOS/pull/409/files#diff-2dad4c3109cff081ead9259c8d910397R279

**Comment by TroubleMakerBen** [comment]
Please use the following translations...in general we want "Fieldname (required)", e.g. `Password (Required)`.


```
EN: $FieldName (Required)
ZH: $FieldName (必填)
RU: $FieldName (Обязательное поле)
PT: $FieldName (Necessário)
IT: $FieldName (Richiesto)
FR: $FieldName (Requis)
ES: $FieldName (Obligatorio)
DE: $FieldName (Erforderlich)
JP: $FieldName (必填)
HR: $FieldName (Obavezno)
HU: $FieldName (Szükséges mező)
NL: $FieldName (Verplicht)
FA: (مورد نیاز) $FieldName 
NB: $FieldName (Påkrevd)
```

File: Resources/ja.lproj/HockeySDK.strings (hunk 1 above)
Comment (erychagov) [inline Resources/ja.lproj/HockeySDK.strings]: @Huan9 @TroubleMakerBen 
Please, find translations for key 'HockeyAuthenticationViewControllerPasswordPlaceholder' for HZ and JP. Translations are different. Is it a mistake?

**Comment by erychagov** [comment]
Restart build

**Pull request merged**